k8s/gcp/gpd-disks/vars.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
this seems to be new file.. should we include this in other ymls?

Reply:
EBS disks attach has this already, introduced one for GPD